src/pattern-library/components/LibraryNext.js

```diff
@@ -72,25 +77,29 @@ function Code({ content, size, title }) {
   const codeMarkup = useMemo(() => jsxToHTML(content), [content]);
 
   return (
-    <figure className="space-y-2">
-      <div
-        className={classnames(
-          'unstyled-text bg-slate-7 text-color-text-inverted p-4 rounded-md',
-          { 'text-sm': size === 'sm' }
+    <figure className="space-y-2 min-h-0 h-full">
```

Comment:
Is the `min-h-0` here for overriding `min-height: auto` if this element happens to be positioned in a flex container?

Reply:
Yep yep yep